contracts/priv/unity.yml

```diff
@@ -966,6 +966,34 @@ paths:
         default:
           description: Unexpected error
           $ref: '#/components/responses/ServerError'
+  /api/v2/quartz/auth/connection:
+    get:
+      operationId: GetConnection
+      tags:
+        - Connection
+      parameters:
+        - $ref: '#/components/parameters/TraceSpan'
```

Comment:
For my own enlightenment: What does this line do?

Reply:
I am not sure what TraceSpan does but I think we pass some meta info in headers through this.

Reply:
I don't think there's any way to specify if a route is authenticated.